lib/src/includes/signed_video_common.h

```diff
@@ -21,6 +21,19 @@
 
 #ifndef __SIGNED_VIDEO_COMMON_H__
 #define __SIGNED_VIDEO_COMMON_H__
+#ifndef HAS_ONVIF
+// Define a fallback version if ONVIF is missing
+typedef enum {
```

Comment:
This cannot be in a public header file. Initially it can be placed in sv_internal.h, but eventually we might use a sv_onvif.h file to put all ONVIF related stuff.